static/game/hero.js

```diff
@@ -20,6 +24,7 @@ export class Hero extends Creature {
   drag = 0.1; // movement deceleration
   invincibilityFrames = 0;
   invincibilityFramesMax = 100;
+  positronParts = 0;
```

Comment:
There should be a better way to keep track of this, especially for when we add more items. Maybe keep a map of item name to number of parts? Also it would be nice if there was a way to see how many parts you have